src/cargo/util_schemas/mood.rs

```diff
@@ -0,0 +1,3 @@
+//! Low-level Cargo format schemas
```

Comment:
I am in a good mood to see this typo 😆

Reply:
I had thought I had already made a file ...